src/components/templates/PartyMap/components/Map/MapRoom.tsx

```diff
@@ -80,9 +81,10 @@ export const MapRoom: React.FC<MapRoomProps> = ({
 
   const [play] = useCustomSound(room.enterSound, { interrupt: true });
   const selectRoomWithSound = useCallback(() => {
+    if (!isLabeled) return;
     play();
     selectRoom();
-  }, [play, selectRoom]);
+  }, [play, selectRoom, isLabeled]);
```

Comment:
This will prevent the room / portal from being interacted with, which definitely isn't something we want to do here. We only want to ensure the label isn't shown on mouseover.

Please revert these changes.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1632/commits/274b6319f73d8b047099bb23ee6d33d8bda5ada2